Makefile.include

```diff
@@ -475,7 +475,7 @@ endif
 
 # set some settings useful for continuous integration builds
 ifeq ($(RIOT_CI_BUILD),1)
-    RIOT_VERSION ?= buildtest
+    RIOT_VERSION ?= $(shell git describe --abbrev=0)
```

Comment:
so so this *almost* works - on CI this prints

    main(): This is RIOT! (Version: 2017.04-devel)

Reply:
I guess thats the latest tag, the CI has. I think, unless your tests rely on it, we can life with it.

Reply:
I just adapted the test accordingly

Reply:
But this is set to a static string for reasons:

1. cacheability, so files don't need to get re-built (and pollute cache) just because the branch is 
2. reproducability, so  otherwise identical binaries stay identical. This is e.g., useful for size comparisons, rules out weird stuff like alignment changes on CM7, and allows the test cache to work. So this needs to stay static.

Reply:
(not sure if this was meant just for testing)

See the result on ccache, this PR's build took "an hour", vs <40min for the ones before. The effect on the test result cache would be even larger (it would practically disable the test result cache).

Reply:
hm but this is only changing once a release (or actually never as we found out that it will always return `2017.04-devel`)

Reply:
Maybe, if the macro should stay static, use 0000.00 statically, here instead of relying on Murdock to never pull any tags in the future (because there is some far too diligent CI admin e. g.)?

Reply:
I don't like forcing the riot version string define into some format that is based on git or anything.
Linux doesn't do that either, they have manual version defines.

Reply:
> I don't like forcing the riot version string define into some format that is based on git or anything.
> Linux doesn't do that either, they have manual version defines.

Welp, setting it to

```suggestion
    RIOT_VERSION ?= 0000.00
```

Would be independent of git :-).